source/render/WorldConstants.hx

```diff
@@ -0,0 +1,5 @@
+package render;
+
+class WorldConstants {
+	public static final HEIGHT_RATIO:Float = 0.5;
```

Comment:
This is just based on Erik's art so far -- A circle on flat ground is half as high as it is wide. So this constant helps us build ranges properly.